src/test/java/org/tarantool/jdbc/JdbcResultSetMetaDataIT.java

```diff
@@ -294,6 +297,7 @@ public void testNotSignedColumns() throws SQLException {
     @Test
     @DisplayName("returned numeric column types")
     public void testColumnsNumericTypes() throws SQLException {
+        assumeServerVersionLessThan(testHelper.getInstanceVersion(), ServerVersion.V_2_2);
```

Comment:
Is it worth to enable this test on 2.2+ too? Maybe just copy-paste it with NUMBER column type?

Reply:
> Support 'number' on 2.2.1 and above. Disable float/real/double on 2.2.0 and above.

so maybe 2.2.1+?

Reply:
See the answer above. Now I see testColumnsNumericTypes() and its pair testColumnsNumberTypeAliases(). Seems okay.